src/models.py

```diff
@@ -30,9 +25,15 @@ class PaginationInfo(BaseModel):
   total_pages: int
 
 
+class FacetInfo(BaseModel):
+  mapping_justification: dict
+  predicate: dict
```

Comment:
best name the facets by their actual sssom model names, i.e. predicate_id

Reply:
Ok, done.